social_distribution/author_manager/urls.py

```diff
@@ -6,9 +6,8 @@
 urlpatterns = [
     path('signup/', views.sign_up, name ="sign_up"),
     path('login/', views.sign_in, name='login'),
-    path('logout/', auth_views.LogoutView.as_view(next_page='author_manager:login'), name='logout'),
-    path('home/', views.home, name='home'),
+    path('', views.home, name='home'),
+    path('logout/', views.sign_out, name='logout'),
     path('authors/<uuid:id>/',  ProfileAPI.as_view(), name='profile'),
```

Comment:
@thihangng I decided to chop off the uuid because some urls have 2 IDs. You might want to change this to `<str:id>`
The new UUIDs format will just be 8 bits instead of the usual 32 bits.

Reply:
sounds good to me!